dynomitemanager/src/main/java/com/netflix/dynomitemanager/sidecore/utils/FifoQueue.java

```diff
@@ -25,20 +25,26 @@
 
     public FifoQueue(int capacity)
     {
-        super(new Comparator<E>()
+    	super(new Comparator<E>()
         {
             @Override
             public int compare(E o1, E o2)
             {
                 return o1.compareTo(o2);
             }
         });
+    	validateCapacity(capacity);
```

Comment:
For validation... IF we dont do it It will fail. So as a Good practice I added the validation there so If someone does not pass it we can easy spot whats wrong. On unit tests I always pass wrong values and invalid parameters to make sure the code does not break. Here was breaking that's why I did it.